modules/library/referencing/src/main/java/org/geotools/geometry/GeometryBuilder.java

```diff
@@ -424,8 +442,8 @@ public Solid createSolid(SolidBoundary boundary) throws MismatchedReferenceSyste
 		return getPrimitiveFactory().createSolid(boundary);
 	}
 
-    public SurfaceBoundary createSurfaceBoundary(PointArray points ) throws MismatchedReferenceSystemException, MismatchedDimensionException {
-        Curve curve = createCurve( points );
+    public SurfaceBoundary createSurfaceBoundary(PointArray points) throws MismatchedReferenceSystemException, MismatchedDimensionException {
+    	Curve curve = createCurve( points );
```

Comment:
No tabs in the source code, please. Please see the [style guide](http://docs.geotools.org/latest/developer/conventions/code/style.html). We also provide an Eclipse ```formatter.xml```.